homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -176,8 +172,14 @@ def __init__(
         else:
             self._attr_native_unit_of_measurement = description.unit_metric
 
-    @property
-    def native_value(self) -> StateType:
-        """Return the state."""
-        state = getattr(self._vehicle.status, self.entity_description.key)
-        return cast(StateType, self.entity_description.value(state, self.hass))
+    @callback
+    def _handle_coordinator_update(self) -> None:
```

Comment:
As we can see it's often more succinct to implement the entity state property than overriding `_handle_coordinator_update` in coordinator entities.